app/src/main/java/com/animal_villa/dao/ICharacterListDAO.kt

```diff
@@ -1,4 +1,7 @@
 package com.animal_villa.dao
 
-class ICharacterListDAO {
+import androidx.room.Dao
+
+@Dao
+interface ICharacterListDAO {
```

Comment:
This was recommended in a previous code review.  When that happens, only first in wins.